.github/workflows/build.yml

```diff
@@ -10,6 +10,9 @@ on:
   schedule:
     - cron: "0 0 * * 0"
 
+# Only one deploy at a time
+concurrency: prod-deploy-group
```

Comment:
I think that makes a lot of sense, let me make that update.

Reply:
Thanks Brian!

Reply:
Do you feel we should also add the `cancel-in-progress: true` as well? This would mean that only the final merged PR would fully run/deploy if there were back to back merges. 

```
deploy:
    needs: test
    concurrency: 
      group: prod-deploy-group
      cancel-in-progress: true
    runs-on: ubuntu-latest
...
```

Reply:
I'm hesitant about `cancel-in-progress: true` since I'm not sure how firebase handles things if we're in the middle of a `firebase deploy` and all of a sudden it gets cancelled. At the end of the day, as long as we keep deploys in order, I'm okay with back to back deploys to go along with the back to back merges. That would probably help us more easily track down deploy issues too if there were problems.

Reply:
Great thinking!